README.md

```diff
@@ -1,10 +1,13 @@
 # Golem Token and Crowdfunding contracts
 
+## Contracts
+GNT contract is defined in contracts/Token.sol. All other contracts are there for testing purposes only.
+
 ## Testing
 
-Testing requires Python and following packages: pyetherem, py.test, hypothesis
+Testing requires Python and following packages: pyetherem, py.test
 
-    pip install ethereum pytest hypothesis
+    pip install ethereum pytest
```

Comment:
Maybe `pip install -r requirements.txt`? `pip install ethereum` will not install the correct version.